semgrep_output_v1.atd

```diff
@@ -787,6 +787,13 @@ type api_scans_findings <ocaml attr="deriving show"> = {
    cai_ids: string list;
 }
 
+type finding_hashes <ocaml attr="deriving show"> = {
```

Comment:
nit: wording confuses me a bit, is this a hash or plural of hashes? also, this seems to be the information that would go into the hash, rather than the hash itself, if im not mistaken.

Reply:
i now understand this and below as hashes _of the things they are named after_, hence the name `finding_hashes` for the type. it seems i was being silly :)